changelogs/7.0.asciidoc

```diff
@@ -0,0 +1,21 @@
+[[release-notes-7.0]]
+== APM Server version 7.0 alpha
```

Comment:
`alpha` added to make it clear from overview pages that all releases have been alpha.  To be changed to beta when appropriate and then remove altogether at release time.